composer.json

```diff
@@ -6,6 +6,9 @@
     "autoload": {
         "psr-4": { "": "src/" }
     },
+    "autoload-dev": {
+        "psr-4": { "Tests\\": "tests/" }
```

Comment:
Well, if you define multiple bundles, this would make it harder to follow the structure (you would have to change the structure of the testsuite)